src/streamlit_extras/app_logo/__init__.py

```diff
@@ -29,9 +29,12 @@ def add_logo(logo_url: str, height: int = 120):
             [data-testid="stSidebarNav"] {{
                 background-image: {logo};
                 background-repeat: no-repeat;
-                padding-top: {height - 40}px;
+                padding-top: {height + 30}px;
```

Comment:
I think the restyling of the sidebar must have changed how high it goes, or maybe I never tested scrolling with a long list, but this prevents the list of pages from overlapping the logo